central/main.go

```diff
@@ -401,7 +402,7 @@ func servicesToRegister() []pkgGRPC.APIService {
 
 	if features.VulnMgmtReportingEnhancements.Enabled() {
 		// TODO Remove (deprecated) v1 report configuration service when Reporting enhancements are enabled by default.
-		servicesToRegister = append(servicesToRegister, reportConfigurationServiceV2.Singleton())
+		servicesToRegister = append(servicesToRegister, reportConfigurationServiceV2.Singleton(), reportService.Singleton())
```

Comment:
This registers the old service. You have to register `v2` service that is introduced through this PR.